README.md

```diff
@@ -190,13 +190,12 @@ To learn more, see:
     exports.handler = signalFxLambda.wrapper((event, context, callback) => {
       ...
       // to send custom event:
-      signalFxLambda.helper.sendCustomEvent('Custom', {functionName: context.functionName}, {description: 'Custom event'})
-         .then(() => callback(null, 'Done'));
+      signalFxLambda.helper.sendCustomEvent('Custom', {functionName: context.functionName}, {description: 'Custom event'});
       
       // to transform & forward CloudWatch event:
-      signalFxLambda.helper.sendCloudWatchEvent(event)
-         .then(() => callback(null, 'Done'))
-      
+      signalFxLambda.helper.sendCloudWatchEvent(event);
+         
+      callback(null, 'Done');
```

Comment:
Why was `.then` removed? Wouldn't this possibly result in the event being lost if the function stopped before the event was exported successfully?